geomstats/exceptions.py

```diff
@@ -0,0 +1,5 @@
+"""Geomstats custom exceptions."""
+
+
+class AutodiffNotImplementedError(RuntimeError):
+    """Raised when autodiff is not implemented."""
```

Comment:
I prefered the RuntimeError option, which seems less "over-engineering", but there might be something I am missing there?

Reply:
see above.